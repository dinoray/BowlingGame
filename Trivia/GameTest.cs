using System;
using System.IO;
using ApprovalTests;
using ApprovalTests.Reporters;
using NUnit.Framework;

namespace BowlingDino
{
    [TestFixture]
    [UseReporter(typeof(DiffReporter))]
    class GameTest
    {
        [Test]
        public void Strike()
        {
            var game = new Game();

            game.Hit(10);
            
            Assert.Equals(game.getResult(),"Strike");
        }
    }
}
