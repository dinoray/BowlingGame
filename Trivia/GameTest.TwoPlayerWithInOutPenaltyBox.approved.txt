John was added
They are player number 1
Mary was added
They are player number 2
John is the current player
They have rolled a 5
John's new location is 5
The category is Science
Science Question 0
Answer was correct!!!!
John now has 1 Gold Coins.
Mary is the current player
They have rolled a 4
Mary's new location is 4
The category is Pop
Pop Question 0
Question was incorrectly answered
Mary was sent to the penalty box
John is the current player
They have rolled a 4
John's new location is 9
The category is Science
Science Question 1
Answer was correct!!!!
John now has 2 Gold Coins.
Mary is the current player
They have rolled a 2
Mary is not getting out of the penalty box
John is the current player
They have rolled a 6
John's new location is 3
The category is Rock
Rock Question 0
Answer was correct!!!!
John now has 3 Gold Coins.
Mary is the current player
They have rolled a 9
Mary is getting out of the penalty box
Mary's new location is 1
The category is Science
Science Question 2
Answer was correct!!!!
Mary now has 1 Gold Coins.
John is the current player
They have rolled a 2
John's new location is 5
The category is Science
Science Question 3
Answer was correct!!!!
John now has 4 Gold Coins.
Mary is the current player
They have rolled a 1
Mary is getting out of the penalty box
Mary's new location is 2
The category is Sports
Sports Question 0
Answer was correct!!!!
Mary now has 2 Gold Coins.
